### PR TITLE
Metastable Flagship

### PR DESCRIPTION
**Pull request by Temoffy**
## About the PR
Add a new ship, the Metastable. It started as a standard Nanotransen station before it was compressed to be more mobile and cheaper to produce. Some scientists don't think it could be compressed that far without fundamentally changing it's state of matter, making a new metastable material. It serves any ships in its vicinity with anything that they might need, processing raw materials brought to it into any number of things.

## Why / Balance
Most fleets use the Crescent as a flagship; but it is expensive and clunky to use. This provides a medium sized alternative flagship that is less luxurious and has less features, but should hopefully prove more useful to small fleets or groups.

## Technical details

yml

## Media

- [ X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added the TC Metastable, a medium-sized station-ship equipped with at least the basics of every department. Recommended to use in conjunction with other ships for any raw materials.

### CODE REVIEW

**Comment by terezi-station** [comment]
nice, you got it. still some comments
1) all (probably not but all the ones i make) windows are floored with plating and all have grilles, just looks nicer
2) Sec is cramped, as well as the dorms, while the bathrooms are rather luxurious, i think removing the hallway above bridge, as in squishing it out of existence by bringing the above wall down, would give sec some more space, and the dorms could have a second pass in response, and jani can find any random home, maybe splitting up that one wide hallway to the airlock below atmos
3) the white tiles with a colored border outside med and sci are cool and look good, the gapped squares that ring them throw them off a lil and can be removed, the little "patio" is enough, and a good use of decals, the med + especially =)
4) chem is a little cramped and maybe a desk to the outside would be nice, to give chems to botany, salv, etc.
5) thruster placement is a little strange, but doesnt matter at all so change it or dont idc

overall, great first outing, talent displayed.

**Comment by Temoffy** [comment]
a few things you noted pertain to rather intentional features (mostly just the decal placement[which I did notice an error or two in] and the hallway above command).

I could be wrong, but I'm going to assume that I may not have made it clear what the design is. The whole ship is as condensed a version of Meta as I could reasonably do while preserving most functionality. What are your thoughts on this? Do you think I ought discard the resemblance in favor of functionality?

I'm going to rework dorms and a few other things you noted while we chat.

**Comment by Temoffy** [comment]
current state

**Comment by Carmell0** [comment]
Maybe  add a more physical treatment room? I don’t really see a place with med beds and medkits, and there’s a fair bitty open space outside med.

**Comment by Temoffy** [comment]
Medkits in the locker, I'll throw a rollerbed in.

**Comment by IData-RedactedI** [comment]
Unless I'm mistaken it looks like the West Side has one less thruster than the other sides.
Overall though it looks very nice

**Comment by Temoffy** [comment]
There is a thruster that isn't powered, thanks for the catch.

**Comment by Carmell0** [comment]
Would like to note the lack of vending machines, is that intentional or did you just forgor

**Comment by Temoffy** [comment]
That is intentional, every one costs 1k but I haven't seen a vender on a ship that has enough in it to meet the 1k mark. As a bonus it lets me save on space and make the loadout for the departments even more minimal.

**Comment by Cheackraze** [comment]
as a note, we want to actually limit the availability of snack/drink vending machines on ships in order to reduce the sources of non-chef cooked food

**Comment by Cheackraze** [comment]
this is great, i only saw 2 things. The gamer loot (CE tool belt and recharging power cells) and east most maints airlock door was unpowered. I decided to fix those things since they were so incredibly minor, in order to get this in!

**Review by Cheackraze — APPROVED** [review]

File: Resources/Maps/Shuttles/metastable.yml [hunk 1]
```diff
@@ -0,0 +1,12592 @@
...
+  - uid: 1195
+    components:
+    - flags: InContainer
+      type: MetaData
+    - parent: 120
+      type: Transform
+    - canCollide: False
+      type: Physics
+    - type: InsideEntityStorage
+- proto: EncryptionKeyCommand
```
Comment (dvir001) [inline Resources/Maps/Shuttles/metastable.yml:4846]: Sigh

**Comment by Temoffy** [comment]
did you remove the drills or did I forget to add them?

**Pull request merged**